central/detection/service/service_impl.go

```diff
@@ -169,6 +170,23 @@ func (s *serviceImpl) DetectBuildTime(ctx context.Context, req *apiV1.BuildDetec
 	}, nil
 }
 
+func (s *serviceImpl) getNetworkPoliciesForDeployment(deployment *storage.Deployment) (*augmentedobjs.NetworkPoliciesApplied, error) {
+	// If the deployment does not contain a clusterId we assume there are no NetworkPolicies
```

Comment:
I'm missing some context I feel like, so got some questions:
Does this really make sense for the `roxctl` check?
We are calling `DetectDeployTimeFromYAML` which will create a `storage.Deployment` out of the YAML file [here](https://github.com/stackrox/stackrox/blob/c46517dc0312d6b600b15581abd0f1321fe47ed4/central/detection/service/service_impl.go#L216). The `cluster ID` will be _always_ empty for this deployment, since it will never have a cluster ID set, so why print the warning conditionally, if the condition will be always met?

Reply:
We are planning to enhance this feature to be able to get alerts on `NetworkPolicies` (e.g. asking the user to provide the cluster where the deployment will be deployed). I can remove the `if` and just return the `augmentedobj` for now; the point of the `if` was to illustrate future steps.